packages/gatsby-recipes/package.json

```diff
@@ -3,58 +3,70 @@
   "description": "Core functionality for Gatsby Recipes",
   "version": "0.1.57",
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
-  "main": "dist/index.js",
   "bugs": {
     "url": "https://github.com/gatsbyjs/gatsby/issues"
   },
   "dependencies": {
-    "@babel/core": "^7.10.3",
-    "@babel/generator": "^7.10.3",
-    "@babel/helper-plugin-utils": "^7.10.3",
-    "@babel/plugin-transform-react-jsx": "^7.10.3",
-    "@babel/standalone": "^7.10.3",
-    "@babel/template": "^7.10.3",
-    "@babel/types": "^7.10.3",
-    "@graphql-tools/schema": "^6.0.11",
-    "@graphql-tools/utils": "^6.0.11",
-    "@hapi/hoek": "8.x",
+    "@babel/core": "^7.9.6",
+    "@babel/generator": "^7.9.6",
+    "@babel/helper-plugin-utils": "^7.8.3",
+    "@babel/plugin-proposal-optional-chaining": "^7.9.4",
+    "@babel/plugin-transform-react-jsx": "^7.9.4",
+    "@babel/standalone": "^7.10.2",
+    "@babel/template": "^7.8.6",
+    "@babel/types": "^7.9.6",
+    "@emotion/core": "^10.0.14",
+    "@emotion/styled": "^10.0.14",
+    "@graphql-tools/schema": "^6.0.14",
+    "@graphql-tools/utils": "^6.0.14",
+    "@hapi/hoek": "8.x.x",
     "@hapi/joi": "^15.1.1",
-    "@mdx-js/mdx": "^1.6.6",
-    "@mdx-js/react": "^1.6.6",
-    "@mdx-js/runtime": "^1.6.6",
-    "acorn": "^7.3.1",
+    "@mdx-js/mdx": "^2.0.0-next.4",
```

Comment:
This breaks tests for gatsby-plugin-mdx, because it requires different version.

Reply:
@johno fixed tests in https://github.com/gatsbyjs/gatsby/pull/26143